com.unity.shadergraph/Editor/Drawing/Inspector/WindowDockingLayout.cs

```diff
@@ -6,7 +6,7 @@
 namespace UnityEditor.ShaderGraph.Drawing
 {
     [Serializable]
-    class WindowDockingLayout
+    public class WindowDockingLayout
```

Comment:
This should be internal, not public. Otherwise we need an API review and public docs.